student-cvs/Josh-19-3875/index.html

```diff
@@ -0,0 +1,91 @@
+    <title>Hello Devs! 💻 </title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header>
+        <h2>WELCOME DEVS - JoshArfaDev</h2>
+        <nav>
+            <ul>
+                <li>Inicio</li>
+                <li>Acerca</li>
+                <li>Portfolio</li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <hr>
+        <h1>Hello There!!</h1>
+        <h2>Here's someone who likes coding and making things with code 🖥️</h2>
+        <article>
+
+            <h3>This is Joshua Acevedo, I'm a web develper who always tries to be a professional dev with my tasks</h3>
+            <code
+                style="background-color: rgb(44, 147, 244); border-radius: 0.3rem; padding: 0.5rem;">Hello World, welcome to my website space!</code>
+
+        </article>
+
+    </main>
+    <br>
+    <section>
+        <ul>
+            <li>I've done a few things with different programing languages📄 such as HTML, Css, JavaScript, Php, design
+                frameworks, frameworks or libraries, databases, Git⭐ ...</li>
+            <li>I've finished a few courses as well and also I've done many things with those knowleges I got.</li>
+            <li>Rightnow I'm studying my major (system engineering) at Mariano Gálvez 🎒</li>
+        </ul>
+    </section>
+
+    <section>
+        <h2 class="contacto-h2">Contact me!</h2>
+        <form action="/server" method="get">
+            
+            <div class="form-group">
+                <label for="name">Full name</label>
+                <input name="name" id="name" type="text" placeholder="Your full name"  required>
+            </div>
+
+            <div class="form-group two-columns">
+                <label class="move" for="email">Email</label>
+                <input name="email" id="email" type="email" placeholder="Your email"  required>
+                <label for="type">City</label>
+                <select name="type" id="type" required>
+                    <option disabled value="" selected>Choose one</option>
+                    <option value="newyork">NewYork</option>
+                    <option value="washington">Washington</option>
+                    <option value="texas">Texas</option>
+                </select>
+            </div>
+
+            <div class="form-group">
+                <label for="description">Describe your project</label>
+                <textarea name="description" id="description" rows="5"></textarea>
+                <div class="help">You can explain with more details what you want.</div>
+            </div>
+
+            <button id="send">Enviar</button>
+
+        </form>
+    </section>
+
+    <footer>
+        <a href="https://josharfdev.github.io/portfolio-react/">Visit my website 🗿</a>
+        <a href="https://github.com/joshArfDev">Visit my Github Profile 🥷</a>
+        <a href="(https://twitter.com/JoshArfa">Visit my twitter 👌</a>
+        <a href="www.linkedin.com/in/joshuaarfa">Visit my linkedin 🧑‍💼</a>
```

Comment:
este link parece no funcionar al hacer click.
creo que el que si funciona seria este:
`https://www.linkedin.com/in/joshuaarfa/`

Reply:
Gracias por todo el feedback crack

Reply:
super, espero que te sirva! 👌